src/App.tsx

```diff
@@ -1,13 +1,27 @@
-import React from "react"
-import CameraCard from './components/CameraCard/CameraCard'
+import CameraCard from "./components/CameraCard/CameraCard";
 
 function App() {
-
   return (
-    <div>
-      <CameraCard />
+    <div className="h-screen bg-[#18a1ad]">
```

Comment:
Podras añadir el estilo a el tailwind config. Algo asi
```suggestion
    colors: {
     'custom-bg': '#18a1ad',
   }
```